main/windows/kap-window.ts

```diff
@@ -162,9 +165,9 @@ export default class KapWindow<State = any> {
 
   // Use this around any call that causes:
   // TypeError: Object has been destroyed
-  private readonly executeIfNotDestroyed = (callback: () => void) => {
-    if (!this.browserWindow.isDestroyed()) {
-      callback();
-    }
-  };
+  // private readonly executeIfNotDestroyed = (callback: () => void) => {
+  //   if (!this.browserWindow.isDestroyed()) {
+  //     callback();
+  //   }
+  // };
```

Comment:
Intention to leave it like this?

Reply:
This was throwing an error that it was unused but I wasn't sure if it'd get used in the future